Engine/source/sim/netGhost.cpp

```diff
@@ -1195,6 +1195,9 @@ NetObject *NetConnection::resolveObjectFromGhostIndex(S32 id)
 
 S32 NetConnection::getGhostIndex(NetObject *obj)
 {
+   if (!obj)
```

Comment:
Every call to this method I can see seems to check if obj is null. Maybe it would be a better idea to have an AssertFatal in here. Either that or change every instance of this call so a redundant check isn't performed.